src/components/vote/VoteItem.js

```diff
@@ -1,12 +1,39 @@
 import React, { Component } from 'react';
+import { Button, Card, Image } from 'semantic-ui-react';
+import DropdownExampleSelection from './Dropdown';
+import awardIcon from '../../images/grammys.png';
+
```

Comment:
Same comment as above. I think the only thing eslint would have to complain about in this file is switching this to a functional component